### PR TITLE
Add stack's testcase

### PR DESCRIPTION
**Pull request by AgwaB**
details:
stack testcase

- [x] Test case

### CODE REVIEW

File: vm/stack_internal_test.go [hunk 1]
```diff
@@ -16,24 +16,69 @@
...
-import "testing"
+import (
+	"testing"
+
+	"github.com/magiconair/properties/assert"
+)
 
 func TestStack_new(t *testing.T) {
+	stack := newStack()
+	assert.Equal(t, stackMaxSize, cap(stack.items))
```
Comment (boohyunsik) [inline vm/stack_internal_test.go]: Someone(including me) uses t.Fatalf method.
I think it is good to use one method to print error.
Or, is there any reason for using assert?

File: vm/stack_internal_test.go [hunk 2]
```diff
@@ -16,24 +16,69 @@
 
 package vm
 
-import "testing"
+import (
+	"testing"
+
+	"github.com/magiconair/properties/assert"
```
Comment (zeroFruit) [inline vm/stack_internal_test.go]: In the case we decided to use assert lib, then change with stretchr

File: vm/stack_internal_test.go (hunk 1 above)
Comment (AgwaB) [reply vm/stack_internal_test.go]: I used it for for no reason and @junbeomlee says we'll use built-in library as you said. 
so i'll change test methods

File: vm/stack_internal_test.go [hunk 3]
```diff
@@ -16,24 +16,83 @@
...
+	stack := newStack()
+
+	stack.push(item(1))
+	stack.push(item(2))
+	stack.push(item(3))
 
+	if len(stack.items) != stack.len() {
+		t.Fatal("function len is invalid")
+	}
+	assert.Equal(t, len(stack.items), stack.len())
```
Comment (hihiboss) [inline vm/stack_internal_test.go]: Is there a reason to compare sizes again?

File: vm/stack_internal_test.go [hunk 4]
```diff
@@ -16,24 +16,83 @@
...
 }
 
 func TestStack_print(t *testing.T) {
```
Comment (hihiboss) [inline vm/stack_internal_test.go:81]: I think this test is not necessary.

File: vm/stack_internal_test.go (hunk 3 above)
Comment (AgwaB) [reply vm/stack_internal_test.go]: i made a mistake... i fixed it!

File: vm/stack_internal_test.go (hunk 4 above)
Comment (AgwaB) [reply vm/stack_internal_test.go:81]: Just i wanted to check that method!

**Comment by hihiboss** [comment]
goimport error!

**Review by junbeomlee — APPROVED** [review]

**Pull request merged**